corehq/apps/cloudcare/tests/test_esaccessors.py

```diff
@@ -101,6 +101,38 @@ def test_limited_users(self):
                 1
             )
 
+    def test_limited_users_case_insensitive(self):
+        self._send_user_to_es(username='superman')
+        self._send_user_to_es(username='robin', user_data={'login_as_user': 'BATMAN'})
+
+        with patch('corehq.apps.cloudcare.esaccessors._limit_login_as', return_value=True):
+            self.assertEqual(
+                login_as_user_query(
+                    self.domain,
+                    MagicMock(username='batman'),
+                    None,
+                    10,
+                    0
+                ).values_list("username", flat=True),
+                ["robin"]
+            )
+
+    def test_limited_users_partial_match(self):
+        self._send_user_to_es(username='superman')
+        self._send_user_to_es(username='robin', user_data={'login_as_user': 'batman and robin'})
```

Comment:
@orangejenny do you think this is an issue? If the user data contains multiple usernames separated by spaces then it will still match. Should that be permitted / supported?

Reply:
I think that's okay, seems like a feature. It'd be good to document it.

Reply:
Docs updated.